views/forms/CharFilterForm/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
+import {
+  Button,
+  Checkbox,
+  List,
+  ListItem,
+  ListItemText,
+} from '@material-ui/core'
+import { FILTER_CATEGORIES } from '../../../constants'
+
+const ChartFilterForm = ({ initialValues, onSubmit, classes }) => {
+  const [values, setValues] = useState(initialValues)
+  const onChange = (target, filterKey) => {
+    setValues({
+      ...values,
+      [filterKey]: values[filterKey].map((filter) =>
+        filter.name === target.name
+          ? { ...filter, value: target.checked }
```

Comment:
should we be using `target.value` instead of `target.checked`?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/248/commits/147bf8a9965b532f34bb176e1e651f339c9bf387